### PR TITLE
Updating rush, adding namedExport support to loader.

### PR DESCRIPTION
**Pull request by dzearing**

### CODE REVIEW

File: loader-load-themed-styles/src/LoadThemedStylesLoader.ts [hunk 1]
```diff
@@ -25,6 +28,13 @@ export class LoadThemedStylesLoader {
   }
 
   public static pitch(remainingRequest: string): string {
+    const options: ILoadThemedStylesLoaderOptions = loaderUtils.getOptions(this) || {};
+    let exportName: string = 'module.exports';
+
+    if (!!options.namedExport) {
```
Comment (iclanton) [inline loader-load-themed-styles/src/LoadThemedStylesLoader.ts]: >!! [](start = 8, length = 2)

You don't need the `!!`

**Comment by iclanton** [comment]
> const loaderUtils = require('loader-utils');

Is this package on the latest version of `loader-utils`? There have been some compatibility breaks with webpack 2+ #Closed

---
Refers to: loader-load-themed-styles/src/LoadThemedStylesLoader.ts:7 in fd9132e. [](commit_id = fd9132e6f0760ea912acec64ce515349ee66e441, deletion_comment = False)

File: loader-load-themed-styles/README.md [hunk 2]
```diff
@@ -24,18 +24,56 @@ var css = require("@microsoft/loader-load-themed-styles!css!./file.css");
...
+              modules: true,
+              importLoaders: 2,
+              localIdentName: '[name]_[local]_[hash:base64:5]',
+              minimize: false
+            }
+          },
+          {
+            loader: 'postcss-loader',
+
+            options: {
```
Comment (iclanton) [inline loader-load-themed-styles/README.md]: Get rid of the blank line.

File: loader-load-themed-styles/README.md [hunk 3]
```diff
@@ -24,18 +24,56 @@ var css = require("@microsoft/loader-load-themed-styles!css!./file.css");
...
+              plugins: function () {
+                return [
+                  require('autoprefixer')
```
Comment (iclanton) [inline loader-load-themed-styles/README.md]: >require('autoprefixer') [](start = 18, length = 23)

Can't `sass-loader` do autoprefixing? #Closed

File: loader-load-themed-styles/README.md [hunk 4]
```diff
@@ -24,18 +24,56 @@ var css = require("@microsoft/loader-load-themed-styles!css!./file.css");
...
+          }
+        ]
+
 ```
 
 ## Options
 
-Ths loader does not take any options at present.
+
+### namedExport (string, defaults to undefined)
```
Comment (iclanton) [inline loader-load-themed-styles/README.md]: >undefined [](start = 37, length = 9)

Put "undefined" in backtick quotes. #Closed

File: loader-load-themed-styles/README.md (hunk 3 above)
Comment (dzearing) [reply loader-load-themed-styles/README.md]: i don't think it does by default. i may be incorrect #Closed

Comment (iclanton) [reply loader-load-themed-styles/README.md]: Nope - doesn't look like it.

---
In reply to: [124628930](https://github.com/Microsoft/webpack-loaders/pull/12#discussion_r124628930) [](ancestors = 124628930)

**Comment by iclanton** [comment]
:shipit:

**Review by iclanton — APPROVED** [review]
Just move the `.npmrc` to `common/rush/config` and it should be good.

**Pull request merged**